src/components/LoggingInNavBar.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import Radium from 'radium';
+
+const styles = {
+  explanation: {
+    marginTop: 75,
+    marginLeft: 100
+  },
+
+  nav: {
+    position: 'relative',
+    marginTop: 75,
+    height: 100,
+    background: '#1f1f1f',
+    marginBottom: 75
+  },
+
+  logo: {
+    position: 'absolute',
+    left: 60,
+    color: 'white',
+    fontWeight: 100,
+    fontSize: 16,
+    top: 45
+  },
+
+  signUpButton: {
+    position: 'absolute',
+    right: 60,
+    top: 30,
+    background: 'none',
+    fontSize: 14,
+    width: 110,
+    color: '#37d087',
+    fontWeight: '300',
+    height: 40,
+    border: '1px solid rgba(255, 255, 255, .2)',
+
+    ':hover': {
+      border: '1px solid #37d087',
+      cursor: 'pointer',
+      transition: '.3s ease-in-out'
+    }
+  }
+}
+
+@Radium
+class LoggingInNavBar extends React.Component {
+  render() {
+    return (
+      <div>
+        <h2 style={styles.explanation}>Navbar when a user is logging in</h2>
+        <nav style={styles.nav}>
+          <h1 style={styles.logo}>Git-Paid</h1>
+          <button type="button" name="button" style={styles.signUpButton}>sign up</button>
```

Comment:
Will only need the linkedin auth button. No need to differentiate between sign up and sign in.